### ISSUE TITLE
How to debug  ERROR ==> An error occurred while installing Osclass

### ISSUE THREAD

**Issue by bcurran3** [bitnami/bitnami-docker-osclass#39]
### Name and Version

bitnami/osclass:latest

### What steps will reproduce the bug?

- Setup bitnami/osclass:latest on Ubuntu 20.04 behind Traefik 1.7.
- Mapped httpd.conf with SSLEngine off after copy of default from container
- chown 1001 * -R in persistent data

container restarts and puts out the following errors:

> osclass 12:31:18.00 INFO  ==> Starting apache in background
> [Fri May 06 12:31:18.104262 2022] [ssl:warn] [pid 144] AH01909: www.example.com:8443:0 server certificate does NOT include an ID which matches the server name
> [Fri May 06 12:31:18.104697 2022] [ssl:warn] [pid 144] AH01909: www.example.com:443:0 server certificate does NOT include an ID which matches the server name
> osclass 12:31:18.14 INFO  ==> apache started
> osclass 12:31:18.35 ERROR ==> An error occurred while installing Osclass

### What is the expected behavior?

Container continues to run. :)
Display more detailed error message pointing to the problem.

### What do you see instead?

> osclass 12:31:18.00 INFO  ==> Starting apache in background
> [Fri May 06 12:31:18.104262 2022] [ssl:warn] [pid 144] AH01909: www.example.com:8443:0 server certificate does NOT include an ID which matches the server name
> [Fri May 06 12:31:18.104697 2022] [ssl:warn] [pid 144] AH01909: www.example.com:443:0 server certificate does NOT include an ID which matches the server name
> osclass 12:31:18.14 INFO  ==> apache started
> osclass 12:31:18.35 ERROR ==> An error occurred while installing Osclass

### Additional information

FEATURE REQUEST: Configuration instructions to disable SSL for working behind reverse proxy such as Traefik. Possibly a different config file to map added to the container. Alternatively adding sudo to the container would be very useful. Then we could install vim/nano/vi/etc easily in the container and edit config files.

**Comment by rafariossaa** [comment]
Please, [note that the Bitnami team is not actively monitoring GitHub Issues for this project](https://github.com/bitnami/bitnami-docker-osclass#community-supported-solution). This issue will stay open for 20 days to allow the community to contribute, after 15 days without activity the issue will be marked as stale, being closed after 5 days.
If you would like to contribute to this container image by fixing a bug or adding a new feature, feel free to create a PR, the Bitnami team will be happy to review it and provide feedback.

**Comment by bcurran3** [comment]
I noticed there was an update to the container today. Unfortunately it didn't fix my problem. :(

> osclass 12:17:48.82 Welcome to the Bitnami osclass container
> osclass 12:17:48.82 Subscribe to project updates by watching https://github.com/bitnami/bitnami-docker-osclass
> osclass 12:17:48.82 Submit issues and feature requests at https://github.com/bitnami/bitnami-docker-osclass/issues
> osclass 12:17:48.82 
> osclass 12:17:48.82 INFO  ==> ** Starting Osclass setup **
> realpath: /bitnami/apache/conf: No such file or directory
> osclass 12:17:48.85 INFO  ==> Configuring Apache ServerTokens directive
> osclass 12:17:48.88 INFO  ==> Configuring PHP options
> osclass 12:17:48.88 INFO  ==> Setting PHP expose_php option
> osclass 12:17:48.89 INFO  ==> Validating settings in MYSQL_CLIENT_* env vars
> osclass 12:17:48.93 WARN  ==> You set the environment variable ALLOW_EMPTY_PASSWORD=yes. For safety reasons, do not use this flag in a production environment.
> osclass 12:17:48.98 INFO  ==> Restoring persisted Osclass installation
> osclass 12:17:49.02 INFO  ==> Trying to connect to the database server
> osclass 12:17:49.11 INFO  ==> Passing update wizard
> osclass 12:17:49.12 INFO  ==> Starting apache in background
> [Sat May 14 12:17:49.186232 2022] [ssl:warn] [pid 143] AH01909: www.example.com:8443:0 server certificate does NOT include an ID which matches the server name
> [Sat May 14 12:17:49.186548 2022] [ssl:warn] [pid 143] AH01909: www.example.com:443:0 server certificate does NOT include an ID which matches the server name
> osclass 12:17:54.22 INFO  ==> apache started
>

**Comment by bcurran3** [comment]
After some playing and observation I figured out the root cause of **ERROR ==> An error occurred while installing Osclass** and am documenting it here incase anyone else has the problem.

When I setup my container I used the environmental variables OSCLASS_USERNAME and OSCLASS_PASSWORD and set generic defaults (Run from an Ansible script.). Once my container is running, the first thing I do is login and change the admin password from the generic default. The container runs fine and all is good, UNTIL... a restart of the container. Then it fails with **ERROR ==> An error occurred while installing Osclass** at startup presumably because it's trying to update osclass (even if there is no update) using the original admin password. I consider the theory proven as I can install from scratch and restart the container multiple times with no problem until I change the admin password. So my solution is to create with my real password and then remove it from my script after the container is deployed.
